documentation/4.0/content/managing-domains/model-in-image/usage.md

```diff
@@ -204,106 +202,3 @@ For more information see:
 - [Using a JDBC Store](https://www.oracle.com/pls/topic/lookup?ctx=en/middleware/fusion-middleware/weblogic-server/12.2.1.4/perfm&id=STORE-GUID-328FD4C1-91C4-4901-B1C4-97B9D93B2ECE) in _Administering the WebLogic Persistent Store_.
 - [High Availability Best Practices](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/jmsad/best_practice.html#GUID-FB97F9A2-E6FA-4C51-B74E-A2A5DDB43B8C) in _Administering JMS Resources for Oracle WebLogic Server_.
 - [Leasing](https://docs.oracle.com/pls/topic/lookup?ctx=en/middleware/fusion-middleware/weblogic-server/12.2.1.4/jmsad&id=CLUST-GUID-8F7348E2-7C45-4A6B-A72C-D1FB51A8E83F) in _Administering Clusters for Oracle WebLogic Server_.
-
-### Requirements for JRF domain types
```

Comment:
There is a lot of good information in this section and I  do not want to lose it. We might want to remove MII references and move the section as its oun page after Domain and Cluster Resources?  I am open to better suggestions.  @robertpatrick thoughts?

Reply:
@mriccell As much as I wanted to find valuable content that might be usable in the future, I think all of this content has such a MII way of doing things bias that anything below would have to be completely rewritten for the Domain on PV way of doing things.

Reply:
Understood then lets save the content for future docs when we add the enhancement to support it on PV

Reply:
???  I just said that I do not believe there is any content worth saving and now you want to save it anyway?  The content is always retrievable from GitHub so there is really no action needed.